cupy/cuda/cub.pyx

```diff
@@ -282,6 +286,37 @@ def device_csrmv(int n_rows, int n_cols, int nnz, ndarray values,
     return y
 
 
+def device_scan(ndarray x, int op):
+    cdef memory.MemoryPointer ws
+    cdef int dtype_id, x_size
+    cdef size_t ws_size
+    cdef void *x_ptr
+    cdef void *ws_ptr
+    cdef Stream_t s
+
+    if op != CUPY_CUB_CUMSUM and op != CUPY_CUB_CUMPROD:
```

Comment:
Can you rewrite this kind of checks as
`if op not in (CUPY_CUB_CUMSUM, CUPY_CUB_CUMPROD):`

Thanks

Reply:
No problem. This is done in 02059fb. Just curious: is this for readability or performance?

Reply:
readability